ubuntu/snappy-playpen
# jTileDownloader app, an example for snapping .jar files

To test:

```
cd jtiledownloader
snapcraft
sudo install *.snap
jtiledownloader
```

From then on, when the GUI has loaded, choose **Slippy Map Chooser** to select a map area and follow the instructions at https://unav-go.github.io/offline to download the tiles.

![jtiledownloader](https://cloud.githubusercontent.com/assets/1689781/16798445/5615ba30-48ec-11e6-9b75-a2c977fdc070.png)

TODO:
- See [README.md](https://github.com/dplanella/snappy-playpen/blob/eafb53fc3eb7864a3cfd6dc66afc21726fdbf20a/jtiledownloader/README.md)



This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ubuntu/snappy-playpen/170)
